test/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter_test.cc

```diff
@@ -860,6 +860,20 @@ TEST_F(GrpcJsonTranscoderFilterTest, TranscodingUnaryPostWithHttpBody) {
   EXPECT_THAT(request, ProtoEq(expected_request));
 }
 
+TEST_F(GrpcJsonTranscoderFilterTest, TranscodingUnaryGetWithHttpBody) {
+  const std::string path = "/echoRawBody?data=oops%7b";
```

Comment:
Not quite understand the intention of setting data parameter here?  Does it want transcoder to set its value to the HttpBody.data field?  

I don't see the code to copy it to the HttpBody.data field.

Reply:
The code shouldn't write data field in to HttpBody.data because HttpBody field shouldn't be override by query param. This test is to test when there is a query param is named with data, and endpoint wants a raw httpbody request info, transcoder should handle this correctly.